summarize/src/main.rs

```diff
@@ -142,7 +142,11 @@ fn diff(opt: DiffOpt) -> Result<(), Box<dyn Error + Send + Sync>> {
 
     table.printstd();
 
-    println!("Total cpu time: {:?}", results.total_time);
+    _ = writeln!(
```

Comment:
I think you should just exit the process on a broken pipe rather than keep writing data to nowhere.

Reply:
Makes sense -- implemented that in 59876d95f6dac8adaf18331269baf323a83e5b6f

Reply:
> Do you think it should special-case broken pipe to return a visible error otherwise?

Yes, I think it should still show an error and exit when an error other than a broken pipe occurred.

Reply:
Fair enough, implemented in 75fde69f873e090a71dc3eacecb2273cd7585ac9